presets/ragengine/main.py

```diff
@@ -310,9 +310,12 @@ async def persist_index(
 )
 async def load_index(
         index_name: str,
-        path: str = Query(DEFAULT_VECTOR_DB_PERSIST_DIR, description="Path to load the index from"),
+        path: Optional[str] = Query(None, description="Path to load the index from"),
         overwrite: bool = Query(False, description="Overwrite the existing index if it already exists")
 ):
+    # If no path is provided, use the default directory joined with index_name.
+    if path is None:
+        path = os.path.join(DEFAULT_VECTOR_DB_PERSIST_DIR, index_name)
```

Comment:
Default here needed to include index name - by default we persist at /storage/index_name - so we will by default load from there as well (given user doesn't provide path)